app/helpers/application_notifications_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ApplicationNotificationsHelper
+  # Translate the event of an activity to the path of a given notification
+  #
+  # @param [Course::Notification | UserNotification] notification the notification that you want
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8858295/8325347/7c96e516-1a8b-11e5-82b5-ce03fd561c48.png)
Actually I find in the doc they use `,` to separate multi-types.